src/hotspot/share/prims/jvmtiRedefineClasses.cpp

```diff
@@ -3694,6 +3679,19 @@ void VM_RedefineClasses::set_new_constant_pool(
       } // end for each local variable table entry
     } // end if there are local variable table entries
 
+    // Update constant pool indices in the method's method_parameters.
+    int mp_length = method->method_parameters_length();
+    if (mp_length >= 0) {
```

Comment:
This line is better to be:
3684       if (mp_length > 0) {

Reply:
Fixed.